### ISSUE TITLE
 No way to remove label for required field

### ISSUE THREAD

**Issue by radzserg** [formly-js/angular-formly-templates-bootstrap#34]
Not sure is it a bug, but if field is required and label is not set then we still have 

```html
<label for="formly_1_input_email_0" class="control-label ng-binding">*</label>
```

http://www.awesomescreenshot.com/image/383232/ee664a49a14e97762bfaddaccbb6552a

http://angular-formly.com/#/example/other/toggle-required
```javascript
{
        key: 'text',
        type: 'input',
        templateOptions: {
          //label: 'Moehahah',
          placeholder: 'Formly is terrific!'
        },
        validation: {
          show: true
        },
        expressionProperties: {
          'templateOptions.required': 'model.checked'
        }
      }
```

**Comment by kentcdodds** [comment]
This is how it's always worked and to change it would be a breaking change. But I agree with the change and personally consider it a bug. I'd accept a PR that fixes the wrapper that causes this. You'll want to add an `ng-if` to [this line](https://github.com/formly-js/angular-formly-templates-bootstrap/blob/master/src/wrappers/label.html#L2)

**Comment by kentcdodds** [comment]
This has been released as a breaking change in `5.0.0`. Enjoy :-)

**Comment by kentcdodds** [comment]
Check that. I made a mistake. Try `5.0.1`
